src/cairols.ts

```diff
@@ -161,6 +149,8 @@ export async function setupLanguageServer(ctx: Context): Promise<SetupResult | u
       new lc.NotificationType<string>("cairo/corelib-version-mismatch"),
       // eslint-disable-next-line @typescript-eslint/no-misused-promises
       async (errorMessage) => {
+        await ctx.statusBar.setStatus({ health: "error", message: errorMessage });
```

Comment:
```suggestion
        await ctx.statusBar.setStatus({ health: "error", message });
```

Please be consistent